shared/src/main/scala/coop/rchain/shared/Stopwatch.scala

```diff
@@ -17,6 +17,23 @@ object Stopwatch {
       _  <- log(s"$tag [${showTime(m)}]")
     } yield a
 
+  def msTime[F[_]: Sync, A](log: String => F[Unit])(block: => F[A]): F[A] =
+    for {
+      t0 <- Sync[F].delay(System.nanoTime)
+      a  <- block
+      t1 = System.nanoTime
+      m  = Duration.fromNanos(t1 - t0).toMillis
+      _  <- log(s"$m")
+    } yield a
+
+  def nsTimeToLong[F[_]: Sync, A](block: => F[A]): F[(A, Long)] =
```

Comment:
This function should return `FiniteDuration` and caller can easily call `toMillis`.